pact/Pact/Core/Compile.hs

```diff
@@ -217,7 +215,7 @@ evalTopLevel bEnv tlFinal deps = do
       let deps' = M.filterWithKey (\k _ -> S.member (_fqModule k) deps) (_loAllLoaded lo0)
           mdata = ModuleData m deps'
       mSize <- sizeOf SizeOfV0 m -- TOOD: Why do we assume SizeOfV0 here?
```

Comment:
I dont see a reason this has to be `SizeOfV0`. Maybe @jmcardon has some argument for it?